src/systems/filecoin_blockchain/storage_power_consensus/storage_power_actor.md

```diff
@@ -20,11 +20,26 @@ SPC provides a power table abstraction which tracks miner power (i.e. miner stor
 
 An invariant of the storage power consensus subsystem is that all storage in the power table must be verified. That is, miners can only derive power from storage they have already proven to the network.
 
-In order to achieve this, Filecoin delays updating power for new sector commitments until the first valid PoSt in the next proving period corresponding to that sector.
+In order to achieve this, Filecoin delays updating power for new sector commitments until the first valid PoSt in the next proving period corresponding to that sector. (TODO: potential delay this further in order to ensure that any power cut goes undetected at most as long as the shortest power delay on new sector commitments).
```

Comment:
@zx just want to make sure this is what you had understood before. 

cc @nikkolasg

Reply:
If i understand correctly, that means a miner does a sealing, then has to submit a post after that (when he's elected of after the "period" or when he's "surprised"). 
If so:
We can do this, but I don't think it's useful to do so, since a sealing already contains a "post" somehow, so it's already sure the miner have spend that much amount of storage. The next post proves that the miner *kept* that storage for that duration. So after a sealing, the power is already there and proven. Not sure if it is needed to require this additional delay. In short, tt definitely doesn't hurt, but i'm not sure if this is needed.

Reply:
Only keeping storage for a duration should count for power, so has to be in arrears. This is also needed for symmetry with the possible lag in the network detecting that a sector has been dropped.